test/plugin/test_in_monitor_agent.rb

```diff
@@ -350,7 +356,9 @@ def get(uri, header = {})
         "plugin_category" => "input",
         "plugin_id"       => "test_in",
         "retry_count"     => nil,
-        "type"            => "test_in"
+        "type"            => "test_in",
+        "emit_records"    => 0,
+        "emit_size"       => 0,
       }
       expected_test_in_response.merge!("config" => {"@id" => "test_in", "@type" => "test_in"}) if with_config
```

Comment:
Maybe extra test cases are needed to confirm `emit_records` and `emit_size` is incremented correctly.

Reply:
Sure, will do.

Reply:
I'm investigating how to test `emit_records` and `emit_size` with current test driver.
But I found that, with current input test driver, it is hard to test against input metrics because event router is replaced with the following rules:
https://github.com/fluent/fluentd/blob/master/lib/fluent/test/driver/test_event_router.rb
https://github.com/fluent/fluentd/blob/master/lib/fluent/test/driver/base_owner.rb#L49

Reply:
I'm trying to write assertion for input plugin statistics on: https://github.com/fluent/fluentd/pull/3411/commits/ad64150d186d576bfa4a06e4797bd8086ecbe37e

Reply:
But I'm not sure why root agent does not ignite metrics callbacks....